src/template.js

```diff
@@ -119,9 +119,20 @@ function chunkPayload(payload, keys, limit) {
 
 switch (data.method) {
   case 'init': {
+    const pointer = copyFromWindow(INSIGHTS_OBJECT_NAME);
+    if (pointer && pointer !== 'aa') {
```

Comment:
Can you elaborate on what the logic here is doing? I can see the basic docs for `copyFromWindow` and I understand that you're essentially pulling down `window['AlgoliaAnalyticsObject']`, but why would pulling it down also mean it's equal to the string `aa`?

Reply:
This is essentially the same logic we use in [search-insights.js](https://github.com/algolia/search-insights.js/blob/main/lib/_createInsightsClient.ts#L13) and [instantsearch](https://github.com/algolia/instantsearch/blob/master/packages/instantsearch.js/src/middlewares/createInsightsMiddleware.ts#L68) as customers can choose to use a property on the window object other than `aa` for their instance of search-insights.js if they need to.

Assigning `window['AlgoliaAnalyticsObject'] a value like `foobar` would mean the customer can send events using:
```js
foobar('clickedObjectIDsAfterSearch', {
  userToken: 'anonymous-123456',
  authenticatedUserToken: 'user-123456',
  eventName: 'Product Clicked',
  index: 'YourIndexName',
  queryID: 'queryID',
  objectIDs: ['objectID-1'],
  positions: [2],
});
```